nais/frontend/dev-gcp.json

```diff
@@ -0,0 +1,22 @@
+{
+    "ingresses": ["https://k9-punsj-frontend.intern.dev.nav.no"],
+    "accessPolicy": {
```

Comment:
Denne bolken kommer ikke in i naiseratorn

Reply:
tror jeg har fiksa nå?

Reply:
Ser bra ut! Bare å teste, indent er spennende :P